cmd/config/constants.go

```diff
@@ -37,6 +37,7 @@ const (
 	// API sub-system
 	EnvAPIRequestsMax      = "MINIO_API_REQUESTS_MAX"
 	EnvAPIRequestsDeadline = "MINIO_API_REQUESTS_DEADLINE"
+	EnvAPIReadyDeadline    = "MINIO_API_READY_DEADLINE"
```

Comment:
I thought it is READINESS_DEADLINE @krishnasrinivas did you check with @abperiasamy ?

Reply:
since the endpoint is /minio/health/ready, i used "ready" instead of "readiness" - easy to remember. You want readiness? i can change that.

Reply:
The reason is because the endpoint is for readiness checks anyways..